src/components/WebexLocalMedia/WebexLocalMedia.js

```diff
@@ -3,24 +3,29 @@ import PropTypes from 'prop-types';
 import {Spinner} from '@momentum-ui/react';
 
 import {WebexAvatar} from '../';
-import {useMe, useMeeting, useStream} from '../hooks';
+import {useElementDimensions, useMe, useMeeting, useStream} from '../hooks';
 
 import './WebexLocalMedia.scss';
 
 /**
- * Webex Local Media component displays the user's local video
+ * Webex Local Media component displays the user's local video.
  *
- * @param {object} props
- * @returns {object} JSX of the component
+ * @param {string} props.meetingID  ID of the meeting from which to obtain local media
+ * @returns {Object} JSX of the component
  */
 export default function WebexLocalMedia({meetingID}) {
+  const [mediaRef, {width}] = useElementDimensions();
   const {localVideo} = useMeeting(meetingID);
   const {ID} = useMe();
   const videoRef = useStream(localVideo);
+  const noMediaClass = localVideo ? '' : ' no-media';
```

Comment:
I think you need to trim that ` no-media`

Reply:
it's on purpose, so there is a space between class names if it gets added